frontend/src/components/app/App.js

```diff
@@ -1,6 +1,5 @@
 import './App.css';
 import LandingPage from '../landing-page/LandingPage'
```

Comment:
is this leftover code from previous projects? there doesnt seem to be anything in that location